### DISCUSSION TITLE
[Q&A] distributed by hostpath?

### DISCUSSION THREAD

**Discussion by momentforever** [minio/minio#20163] [Q&A] [answered]
+ Whether hostpath mode can be used in the case of distributed deployment？
+ If possible, What strategy is used for redundancy?
+ If I have 23 machines with 12 hard disks each, what is the recommended mode to deploy in K8S?

> THX!

**Comment by harshavardhana** [comment, marked as answer]
Use https://github.com/minio/directpv - no hostpath

> * If possible, What strategy is used for redundancy?
> * If I have 23 machines with 12 hard disks each, what is the recommended mode to deploy in K8S?

Ping us at hello@minio.io for such an analysis.

**Reply by momentforever** [reply]
The upload speed is only 80mb/s in distributed mode but 150mb/s in Hostpath, is this normal?

**Reply by harshavardhana** [reply]
Never run micro benchmarks.

**Reply by momentforever** [reply]
thx!
